metadata-ingestion/src/datahub/ingestion/source/looker/looker_config.py

```diff
@@ -209,12 +209,17 @@ class LookerDashboardSourceConfig(
     )
     extract_independent_looks: bool = Field(
         False,
-        description="Extract looks which are not part of any Dashboard. To enable this flag the stateful_ingestion should also be enabled.",
+        description="Extract looks which are not part of any Dashboard. To enable this flag the stateful_ingestion "
+        "should also be enabled.",
     )
     emit_used_explores_only: bool = Field(
         True,
         description="When enabled, only explores that are used by a Dashboard/Look will be ingested.",
     )
+    include_looker_element_in_platform_instance: bool = Field(
+        False,
+        description="When enabled, platform instance will be added in dashboard and chart urn.",
```

Comment:
please also add a validator that displays a warning if platform_instance is set but `include_platform_instance_in_urns` is unset / false. Also make sure to add a test for that validator